frontend/src/graphql/queries.ts

```diff
@@ -15,3 +15,26 @@ const findAllUserProfiles = async () => {
   return data.users;
 };
 export default findAllUserProfiles;
+
+const joinTownMutation = gql`
+  mutation joinTown($TownJoinRequest: TownJoinRequest!)
+  @rest(type: "Town", path: "/sessions", method: "POST") {
+    isOK
+    response
+    message
+  }
+  `;
+
+const TownJoinRequest = gql`
```

Comment:
This is not needed.
The argument is something that would be passed to joinTown function.